nixos/doc/manual/release-notes/rl-2211.section.md

```diff
@@ -274,6 +274,8 @@ Available as [services.patroni](options.html#opt-services.patroni.enable).
 
 - A new module was added for the Saleae Logic device family, providing the options `hardware.saleae-logic.enable` and `hardware.saleae-logic.package`.
 
+- ZFS module will not allow hibernation by default, this is a safe measure to prevent data loss cases like the ones described at [OpenZFS/260](https://github.com/openzfs/zfs/issues/260) and [OpenZFS/12842](https://github.com/openzfs/zfs/issues/12842).
```

Comment:
I think we should document the option here, for the users that want to disable this behavior even when they understand their consequences.

Reply:
An additional sentence like "Use the `boot.zfs.allowHibernation` option to configure this behaviour."?

Reply:
Yeah, something like this.